medusa/process_tv.py

```diff
@@ -573,7 +573,6 @@ def _process_postponed(self, processor, path, video, ignore_subs):
             self._log('Subtitles check was disabled for this episode in manual post-processing. '
                       'Continuing the post-processing of this file: {0}'.format(video))
 
-        self.postpone_processing = False
```

Comment:
@medariox in my tests this was needed. I didn't understood why but it was needed